packages/appstore/src/components/CFDs/cfd-demo-accounts.tsx

```diff
@@ -137,7 +137,9 @@ const CFDDemoAccounts = ({ isDerivedVisible, isFinancialVisible, current_list }:
                                                 <AccountManager
                                                     has_account={true}
                                                     type={existing_account.market_type}
-                                                    appname={`${account.name} ${title_shortcode}`}
+                                                    appname={`${account.name} ${
```

Comment:
add this condition on `title_shortcode`
```suggestion
                                                                               const title_shortcode = (is_eu || existing_account.platform === CFD_PLATFORMS.DXTRADE) ? '' : non_eu_accounts;
```